Notify/ServiceLayer/NotificationCRUDService/NotificationCRUDService.cs

```diff
@@ -54,5 +57,18 @@ public void Delete(int Id)
             context.Notifications.Remove(notification);
             context.SaveChanges();
         }
+
+
+        public NotificationDetailDTO GetRandomNotification()
+        {
+            var count = context.Notifications.Count();
+            var notifications = context.Notifications.ToList();
```

Comment:
You should never ToList() your DbSet. You can get count, then random number and then use Skip() and First() methods to fetch from your database only one entity that you really need, not all